src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/entity/PrintDetailsBuilder.kt

```diff
@@ -38,7 +38,7 @@ fun buildPrintDetails(
     photoLocation: String = "arn:aws:s3:::source-document-storage/$gssCode/$sourceReference/${UUID.randomUUID()}/" +
         faker.file().fileName("", null, "jpg", ""),
     status: Status = Status.PENDING_ASSIGNMENT_TO_BATCH,
-    batchId: UUID? = UUID.randomUUID()
+    batchId: String? = UUID.randomUUID().toString()
```

Comment:
Minor - I know this only a test builder, but is it worth removing the dashes from the batchId so that it better reflects what will actually be set?
```
batchId: String? = UUID.randomUUID().toString().replace("-", "")
```